nav2_bringup/launch/multi_tb3_simulation_launch.py

```diff
@@ -181,9 +177,6 @@ def generate_launch_description():
     ld.add_action(declare_rviz_config_file_cmd)
     ld.add_action(declare_use_robot_state_pub_cmd)
 
-    # Add the actions to start gazebo, robots and simulations
-    ld.add_action(start_gazebo_cmd)
```

Comment:
Removing gazebo? Wouldn't we have multiple instances of it running if each of the `n` robots starts their own? I thought we did that here so that it didn't get done there so they all shared 1 and we set the gazebo parameters to false within each's launch (e.g. https://github.com/ros-planning/navigation2/blob/7f634a3993bd5e129c5b5b220795df9a09a332bd/nav2_bringup/launch/multi_tb3_simulation_launch.py#L133). I think gazebo needs to be launched here?

Reply:
With `start_gazebo_cmd ` I got this error: `[Err] [Master.cc:95] EXCEPTION: Unable to start server[bind: Address already in use]. There is probably another Gazebo process running.` A reason could be that a gazebo process is already started by `tb3_simulation_launch.py`.

Reply:
You were right. I misunderstood the use of that parameter. Anyway, I'm having some problem: map is not load correctly, so the simulation is not working properly.

Reply:
Did that get worked out?

Reply:
Nope. Still debugging it.

Reply:
OK!